add support for VTE3 based terminals above v0.39

jck opened the pull request.

jszakmeister commented on doc/togglecursor.txt:
```diff
@@ -18,13 +18,14 @@ supported terminals.
-The gnome-terminal application doesn't support changing the cursor via escape
-sequences and is not supported.  On unsupported terminals, Vim's default
-behavior is left unaltered.
+beta or better is required), VTE3 based terminals(including gnome-terminal),
```
You're missing a space between "terminals" and "(including".

jszakmeister commented on plugin/togglecursor.vim:
```diff
@@ -38,8 +38,8 @@ let s:supported_terminal = ''
                 \ || $XTERM_VERSION != ""
-                " \ || $VTE_VERSION != ""
-        " iTerm, xterm, and future VTE based terminals support DESCCUSR.
+                \ || str2nr($VTE_VERSION) >= 3900
```
Nice to see support has finally shown up!  What are you running that you have access to this now?

jck replied:
I use termite (https://github.com/thestinger/termite)

jszakmeister commented:
Would you mind fixing the commit message to capitalize the first letter and add a period to the end?  And it's not really "above v0.39".  It's more like "v0.39 or better", given your conditional, correct?  Thanks!

jck commented:
Yes, it is v0.39 and above. I've fixed the commit message and added a space in the help text.
Feel free to amend my commit if you find any other issues :)

jszakmeister commented:
Looks good!  Thank you.
